infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -704,15 +696,14 @@ public void shutdown() {
     @Builder
     @Getter
     public static class Config {
-        private static final long DEFAULT_CACHE_SIZE = 250_000L;
+        private static final int DEFAULT_CACHE_SIZE = 250_000;
 
         @Default
-        private final long cacheSize = DEFAULT_CACHE_SIZE;
+        private final int cacheSize = DEFAULT_CACHE_SIZE;
 
         public static Config parse(Map<String, Object> opts) {
-            long cacheSize = opts.containsKey("--sequencer-cache-size") ?
-                    Long.parseLong((String) opts.get("--sequencer-cache-size")) : DEFAULT_CACHE_SIZE;
-
+            int cacheSize = (int)(opts.containsKey("--sequencer-cache-size") ?
+            Long.parseLong((String)opts.get("--sequencer-cache-size")) : DEFAULT_CACHE_SIZE);
```

Comment:
why parse into a long and then typecast to an integer?  Can we use Integer.parseInt() instead?